integration-libs/epd-visualization/root/config/epd-visualization-test-config.ts

```diff
@@ -0,0 +1,36 @@
+import { EpdVisualizationConfig } from './epd-visualization-config';
+
+/**
+ * Just used for tests.
+ */
+export function getTestConfig(): EpdVisualizationConfig {
```

Comment:
This should not be a part of public API.
- let's move it to `/integration-libs/epd-visualisation/root/testing`, by convention
- let's not export it in the public api via `index.ts` barrel

Reply:
I have mode this out of the public API and put it in `/integration-libs/epd-visualisation/root/testing`.